0020-bluetooth-payment-protocol/0020-bluetooth-payment-protocol.md

```diff
@@ -0,0 +1,117 @@
+# Bluetooth Payment Protocol
+
+This document specifies a protocol for communicating and authorizing Interledger Payment Requests over Bluetooth (Low Energy). It is primarily intended for use in in-person retail payment contexts.
+
+This protocol may be implemented by Customer devices such as smartphones or very low-power, offline devices such as tapFi's.
+
+## Definitions
+
+- `IPR`: [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md)
+- `Merchant`: The one requesting a payment
+- `Customer`: The one authorizing and paying for payment requests
+- `Connection`: Short time binding between `Merchant` and `Customer`
+- `ILP Account`: An Interledger-enabled account on a ledger
+- `ILP Address`: [Interledger](https://github.com/interledger/rfcs/blob/master/0015-ilp-addresses/0015-ilp-addresses.md) address in the form of `g.us.somebank.youraccount`
+- `Merchant Certificate Authority`: Like CA's for TLS certificates, Merchant CAs sign the certificates of trusted merchants. CAs MUST maintain blacklists of compromised keys
+- `Redemption URI`: URI where the Merchant can submit Customer-authorized IPRs to initiate an ILP payment to their account (Note: the service that implements this MUST store previously redeemed IPRs to avoid duplicate payments)
+
+## GATT Services: BLE Description
```

Comment:
Can you provide a short description of what this means for those unfamiliar with Bluetooth?